SteamKit2/SteamKit2/Steam/Handlers/SteamFriends/SteamFriends.cs

```diff
@@ -57,7 +57,7 @@ internal SteamFriends()
 
 
         /// <summary>
-        /// Gets the local user's persona name.
+        /// Gets the local user's persona name. Can be null before user's initialization.
```

Comment:
I think we need to do a better job of defining exactly when the local user's persona name will be available.

Reply:
Feel free to update PR with a better explanation if appropriate - GitHub recently added a way for repo maintainers to update PRs if contributor enables such option 🙂.